server/src/routes/api/v1/shoeReviewsRouter.js

```diff
@@ -6,6 +6,16 @@ import ReviewSerializer from "../../../serializer/ReviewSerializer.js"
 
 const shoeReviewsRouter = new express.Router({ mergeParams: true })
 
+shoeReviewsRouter.delete("/:id", async (req, res) => {
+  const reviewId = req.params.id
+  try {
+    const deletedReview = await Review.query().findById(reviewId).delete()
+    return res.status(201).json({ deleted: deletedReview })
```

Comment:
delete status should be 204 - 201 indicates something was created, but here we are deleting

we don't need to send back the deleted review, as it now does not exist. Instead, send back something like a message to other developers to let them know it was successful

```js
res.status(204).json({ message: "review successfully deleted" })
```